digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/strategy/DefaultSectionCalculatorStrategy.scala

```diff
@@ -310,6 +310,7 @@ class DefaultSectionCalculatorStrategy extends RoadAddressSectionCalculatorStrat
           val onceConnectLinks = TrackSectionOrder.findOnceConnectedLinks(linksWithoutValues)
           val existsCloserProjectlink = linksWithValues.filter(pl => pl.startAddrMValue < endLinkWithValues.startAddrMValue && pl.id != endLinkWithValues.id)
           if (endPointsWithValues.nonEmpty && onceConnectLinks.nonEmpty && linksWithValues.nonEmpty
+            //TODO comment 2 lines added down below
```

Comment:
Should something be done regarding this TODO-comment?

Reply:
yes it could be deleted